components/Pages/Project/Review/CardReviewSummary.tsx

```diff
@@ -0,0 +1,279 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+import { GrantStory, ReviewMode } from "@/types/review";
+import { PlusIcon } from "@heroicons/react/24/solid";
+import { Button } from "@/components/Utilities/Button";
+import { isAddressEqual } from "viem";
+import { DynamicStarsReview } from "./DynamicStarsReview";
+import { useProjectStore } from "@/store";
+import { useAccount, useSwitchChain } from "wagmi";
+import { useConnectModal } from "@rainbow-me/rainbowkit";
+import toast from "react-hot-toast";
+import { arbitrum } from "viem/chains";
+import { useReviewStore } from "@/store/review";
+import { getBadgeIds } from "@/utilities/review/getBadgeIds";
+import { SCORER_DECIMALS, SCORER_ID } from "@/utilities/review/constants/constants";
+import { getBadge } from "@/utilities/review/getBadge";
+import { ProgressBar } from "./ProgressBar";
+import React, { useEffect, useState } from "react";
+import { useSearchParams } from "next/navigation";
+
+export const CardReviewSummary = () => {
+  const project = useProjectStore((state: any) => state.project);
+  const setIsOpenReview = useReviewStore((state: any) => state.setIsOpenReview);
+  const setActiveBadges = useReviewStore((state: any) => state.setActiveBadges);
+  const setActiveBadgeIds = useReviewStore((state: any) => state.setActiveBadgeIds);
+  const stories = useReviewStore((state: any) => state.stories);
+  const [timestampInterval, setTimestampInterval] = useState<number | null>(null);
+  const [averageScoreReview, setAverageScoreReview] = useState<number | null>(null);
+  const [intervalMessage, setIntervalMessage] = useState<string | undefined>(undefined);
+  const [starRatingFiltered, setStarRatingFiltered] = useState<StarRatingDataProps[]>([]);
```

Comment:
I'd call it `reviewsScoreWithPercentage`

Reply:
what state ?